src/components/NavBar.js

```diff
@@ -1,33 +1,45 @@
-import React, { Component } from 'react';
-import { NavLink } from 'react-router-dom';
-import { Container, Menu, Input } from 'semantic-ui-react';
-import '../styling/Navbar.css';
-import AlertMessage from './AlertMessage';
-import { connect } from 'react-redux';
+import React, { Component } from "react";
+import { NavLink } from "react-router-dom";
+import { Container, Menu, Input } from "semantic-ui-react";
+import "../styling/Navbar.css";
+import AlertMessage from "./AlertMessage";
+import { connect } from "react-redux";
+import UpdateLanguage from './UpdateLanguage';
+import i18n from "../i18n";
 
 class NavBar extends Component {
-  state = { activeItem: 'latest news' }
+  constructor(props) {
+    super(props)
+    this.ReRender = this.ReRender.bind(this);
+  };
```

Comment:
I would need an explanation to: 

1. Why did you remove the `activeItem` state object
2. Why do you use `forceUpdate `?

Reply:
activeItem should not have been removed.  It has been included back in the file.

Reply:
2. By only passing props in order to update the language, it for some reason only lets us change from the initial language to the other one, but not change it back. ForceUpdate is causing the component to rerender without rerendering the entire app.

Reply:
I have a different solution to this problem that involves component state. Please take a look at https://github.com/tochman/gnosis-client/commit/b0550b60dd6465f3965700a42efec308678795f5

Reply:
However, I would argue that we should store the preferred language in the application state.